hs_core/templates/pages/genericresource.html

```diff
@@ -208,12 +210,14 @@ <h4 class="modal-title">Confirm file deletion</h4>
                             {#  Resource specific metadata can't be edited for HIS Referenced Time Series #}
                             {% if cm|resource_type != "HIS Referenced Time Series" or not metadata_form %}
                                 {% if  cm|resource_type != "Generic" %}
-                                    <li class="col-xs-2 col-md-2">
-                                        <a href="#resourceSpecificTab" data-toggle="tab" title="Resource specific">
-                                            <span class="glyphicon glyphicon-folder-close"></span>
-                                            &nbspResource Specific
-                                        </a>
-                                    </li>
+                                    {% if  cm|resource_type|lower != "collection resource" %}
```

Comment:
@hyi We didnt remove the check for Generic resources, we just add another check for Collection resource. So the final logic is:
 if current res is neither Generic nor Collection, then display "resourceSpecificTab" tabpage.